tests/kubernetes/bin/test_paasta_secrets_sync.py

```diff
@@ -40,46 +43,49 @@ def test_main():
             assert e.value.code == 1
 
 
-@pytest.mark.parametrize("namespace", [None, "tron"])
-def test_sync_all_secrets(namespace):
+def test_sync_all_secrets():
     with mock.patch(
         "paasta_tools.kubernetes.bin.paasta_secrets_sync.sync_secrets", autospec=True
     ) as mock_sync_secrets, mock.patch(
         "paasta_tools.kubernetes.bin.paasta_secrets_sync.PaastaServiceConfigLoader",
         autospec=True,
     ):
+        services_to_k8s_namespaces = defaultdict(set)
+        services_to_k8s_namespaces["foo"].add(
+            get_instance_type_to_k8s_namespace()["kubernetes"]
+        )
+        services_to_k8s_namespaces["bar"].add(
+            get_instance_type_to_k8s_namespace()["kubernetes"]
+        )
```

Comment:
might be nice to construct this as a normal dict so that we make sure we're not depending on things that use this data to be expecting a defaultdict